### DISCUSSION TITLE
Wrong data is delivered since two days

### DISCUSSION THREAD

**Discussion by coarsy** [mampfes/hacs_waste_collection_schedule#2206] [Q&A]
After restarting HA yesterday, the data delivered from the service is corrupt. There is no data in the calendar too or only in the far future. It has always worked perfectly over the last few weeks and months. Is there something wrong or broken?

This is my configuration:

`waste_collection_schedule:
    sources:
    - name: app_abfallplus_de
      args:
        app_id: de.k4systems.zakb
        city: Heppenheim
        strasse: xxx
        hnr: xxx`

![bug_waste](https://github.com/mampfes/hacs_waste_collection_schedule/assets/2741933/5539ed04-6afb-4d40-a3aa-5d6fdbed0c50)

Thanks in advantage and best regards,

Chris

**Comment by 5ila5** [comment]
app_abfallplus_de is seeing some weird behavior:

closing this one for #2197
